canonical-websites/www.ubuntu.com
# Improve the summary messaging from the interactive forms

## Done
Improve the summary messaging from the interactive forms and applied the design points to the header of the modal.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /core#get-in-touch
- Fill in the form until you reached the third page which asks for your details
- Open the console and enter `Comments_from_lead__c.value`
- Check that the information is correct and looks better then on live
